src/Components/TabContainer/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import tabStyles from './tab.module.css'
+import tabBodyStyles from './tabBody.module.css'
+import containerStyles from './container.module.css'
+import classnames from 'classnames'
+
+export const TabContainer = ({ tabNames, renderTab, renderBody }) => {
+  const [activeTab, setActiveTab] = React.useState(tabNames[0])
+  const onTabChange = (key) => {
+    setActiveTab(key)
+  }
+
+  return (
+    <div className={containerStyles.container}>
+      <ul className={containerStyles.tabList}>
+        {tabNames.map((name) => {
+          const isActive = activeTab === name
+          return renderTab(name, onTabChange, isActive)
+        })}
+      </ul>
+      <TabBody>{renderBody(activeTab)}</TabBody>
+    </div>
+  )
+}
+
+TabContainer.propTypes = {
+  tabNames: PropTypes.arrayOf(PropTypes.string).isRequired,
+  renderTab: PropTypes.func.isRequired,
+  renderBody: PropTypes.func.isRequired
+}
+
+TabContainer.defaultProps = {
+  tabNames: ['Tab1', 'Tab2'],
+  renderTab: (name, callback, isActive) => (
+    <Tab title={name} tabKey={name} onTab={callback} isActive={isActive} />
+  ),
+  renderBody: (activeTab) => (
+    <>
+      {activeTab === 'Tab1' && <div>Tab 1 is active</div>}
+      {activeTab === 'Tab2' && <div>Tab 2 is active</div>}
+    </>
+  )
+}
+
+export const Tab = ({ title, isActive, onTab, tabKey }) => {
```

Comment:
Why do you need Tab component. There are no tabs in your project description/

Reply:
It looks like copy-paste from the example https://github.com/kse-web-development-practice/example/blob/main/src/Components/TabContainer/index.js